layer/layer_data.cc

```diff
@@ -269,4 +269,16 @@ LayerData::ShaderModuleCreateResult LayerData::CreateShaderModule(
       HashShader(*shader_module, create_info->pCode, create_info->codeSize);
   return {result, hash, start, end};
 }
+
+void LayerData::DestroyShaderModule(
+      VkDevice device, VkShaderModule shader_module,
+      const VkAllocationCallbacks* allocator) {
+  auto next_proc =
+      GetNextDeviceProcAddr(device, &VkLayerDispatchTable::DestroyShaderModule);
+  absl::Time start = absl::Now();
+  next_proc(device, shader_module, allocator);
+  absl::Time end = absl::Now();
```

Comment:
I don't think we have any immediate use case for destruction time. Could we remove this until we find one?